gui/webdisplay/src/RWebWindowsManager.cxx

```diff
@@ -472,7 +474,6 @@ void ROOT::Experimental::RWebWindowsManager::Terminate()
    if (fServer)
       fServer->SetTerminate();
 
-   // use timer to avoid situation when calling object is deleted by terminate
```

Comment:
@linev I don't understand why you are using a timer here. Can you elaborate?

Reply:
I have doubt, that gApplication->Terminate() may call destructor of any object - including RWebWindowManager itself. But if direct call works better - just use it